app/main.py

```diff
@@ -1 +1,87 @@
-# write your code here
+from app.cafe import Cafe, date
+from app.errors import \
+    NotVaccinatedError, \
+    OutdatedVaccineError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    is_vaccinated = True
+
+    for idx, friend in enumerate(friends):
+        try:
+            if "vaccine" not in friend:
+                is_vaccinated = False
+                break
+
+            if friend["vaccine"]["expiration_date"] < date.today():
+                is_vaccinated = False
+                break
```

Comment:
You already have this chech in method `visit_cafe()`. Look like dublication, try to remove it